client/styles/sublayout.scss

```diff
@@ -72,8 +70,7 @@
     .sublayout-main-col {
         flex: 3;
         padding: 0;
-        padding-right: 20px;
-        max-width: calc(100% - #{$sidebar-width});
+        padding: 36px;
```

Comment:
This additional top padding brings the discussion-main section out of alignment with the sidebar, which is probably undesirable.